server/gc/src/main/java/org/apache/accumulo/gc/GCRun.java

```diff
@@ -491,6 +509,10 @@ static void minimizeDeletes(SortedMap<String,GcCandidate> confirmedDeletes,
           lastDirAbs = null;
         }
       }
+      if (progressTimer.hasElapsed(Duration.ofMinutes(1))) {
```

Comment:
This may create a Duration object each time the loop iterates.  Could create a Duration constant for the one minute and use that for this and the other chechs.